packages/hash/frontend/src/components/BlockContextMenu/BlockContextMenu.tsx

```diff
@@ -35,22 +35,22 @@ const MENU_ITEMS: Array<MenuItemType> = [
   {
     key: "delete",
     title: "Delete",
-    icon: <DeleteIcon className={iconStyles} />,
+    icon: <DeleteIcon sx={iconStyles} />,
```

Comment:
Currently `iconStyles` is a tailwind classname - are you planning on updating this to be a valid argument for `sx`? If not, I'd continue passing this as a `className` for now.

Reply:
This is an artifact of me branching off of Teenoh's then-draft PR. Reverting this